content/en/api/rate_limiting/rate_limiting.md

```diff
@@ -16,12 +16,14 @@ Regarding API rate limit policy:
 * Datadog **does not rate limit** on data point/metric submission (see [metrics section][2] for more info on how metric submission rate is handled). Limits encounter is dependent on the quantity of [custom metrics][3] based on your agreement.
 * The rate limit for metric **retrieval** is `100` per hour per organization.
 * The rate limit for event submission is `1000` per aggregate per day per organization. An aggregate is a group of similar events.
-* The rate limit for the [query_batch API][4] and [Log Query API][5] call is `300` per hour per organization. This can be extended on demand.
-* The rate limit for the [graph_snapshot API][6] call is `60` per hour per organization. This can be extended on demand.
+* The rate limit for the [query_batch API][4], [Log Query API][5] [Trace Query API][6] call is `300` per hour per organization. This can be extended on demand.
```

Comment:
```suggestion
* The rate limit for [query_batch API][4], [Log Query API][5], and [Trace Query API][6] call is `300` per hour per organization. This can be extended on demand.
```